config/routes.rb

```diff
@@ -249,6 +249,8 @@
   post 'webhooks/twitter', to: 'api/v1/webhooks#twitter_events'
   post 'webhooks/line/:line_channel_id', to: 'webhooks/line#process_payload'
   post 'webhooks/telegram/:bot_token', to: 'webhooks/telegram#process_payload'
+  get 'webhooks/instagram', to: 'api/v1/webhooks#instagram_verify'
```

Comment:
let's have a dedicated controller for Instagram handling the endpoints. In the future, we will decouple the Twitter events from the webhooks controller as well.

Reply:
Okay.